### PR TITLE
Added cs-office-365 module

### PR DESCRIPTION
**Pull request by ionutmarisca**
Added new **cs-office-365** module containing `GetAuthenticationToken` action.

### CODE REVIEW

File: cs-office-365/src/main/java/io/cloudslang/content/office365/actions/utils/GetAuthorizationToken.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+import static io.cloudslang.content.utils.OutputUtilities.getSuccessResultsMap;
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static org.apache.commons.lang3.StringUtils.defaultIfEmpty;
+
+public class GetAuthorizationToken {
+
+    /**
+     * @param loginType      Login method according to application type
+     *                       Valid values: 'API', 'Native'
+     *                       Default: 'API'
```
Comment (liviu-graur) [inline cs-office-365/src/main/java/io/cloudslang/content/office365/actions/utils/GetAuthorizationToken.java]: Please remove description from here

File: cs-office-365/src/main/java/io/cloudslang/content/office365/actions/utils/GetAuthorizationToken.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                    @Output(value = EXCEPTION, description = EXCEPTION_DESC)
+            },
+            responses = {
+                    @Response(text = SUCCESS, field = RETURN_CODE, value = ReturnCodes.SUCCESS, matchType = COMPARE_EQUAL, responseType = RESOLVED, description = SUCCESS_DESC),
+                    @Response(text = FAILURE, field = RETURN_CODE, value = ReturnCodes.FAILURE, matchType = COMPARE_EQUAL, responseType = ERROR, description = FAILURE_DESC)
+            })
+    public Map<String, String> execute(@Param(value = LOGIN_TYPE, description = LOGIN_TYPE_DESC) String loginType,
+                                       @Param(value = CLIENT_ID, required = true, description = CLIENT_ID_DESC) String clientId,
+                                       @Param(value = CLIENT_SECRET, encrypted = true, description = CLIENT_SECRET_DESC) String clientSecret,
+                                       @Param(value = USERNAME, description = USERNAME_DESC) String username,
```
Comment (liviu-graur) [inline cs-office-365/src/main/java/io/cloudslang/content/office365/actions/utils/GetAuthorizationToken.java]: mark as final

File: cs-office-365/src/main/java/io/cloudslang/content/office365/actions/utils/GetAuthorizationToken.java (hunk 1 above)
Comment (ionutmarisca) [reply cs-office-365/src/main/java/io/cloudslang/content/office365/actions/utils/GetAuthorizationToken.java]: Personally, I would like to keep this description for other developers to see it when opening the action. 👍

File: cs-office-365/src/main/java/io/cloudslang/content/office365/actions/utils/GetAuthorizationToken.java (hunk 2 above)
Comment (ionutmarisca) [reply cs-office-365/src/main/java/io/cloudslang/content/office365/actions/utils/GetAuthorizationToken.java]: Unfortunately, the implementation would require me to add new inputs with different names. Good point, btw ;).

**Pull request merged**